### PR TITLE
DDS:  F5 Distributed Cloud integration (V2.0.0)

### PR DESCRIPTION
**Pull request by madhavpandya-crest**
### What does this PR do?

PR for a integration F5 Distributed Cloud 2.0.0

### Motivation
-- OOTB detection rules JSON would be shared separately with the required teams as a part of separate repository .
-- During the standard attribute remapping, we preserve certain source attributes to ensure compatibility with older versions without causing conflicts. For other fields, we do not preserve them as per suggested best practices. As a result, filters using these standard attributes will populate values from other integrations, following the current datadog behavior.

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)
- [ ] If this PR includes a log pipeline, please add a description describing the remappers and processors. 

### Additional Notes

- The system currently does not monitor DDoS events. This functionality will be added once relevant event data becomes available.
- ~The pipeline is encountering failures due to source conflicts. This issue needs to be resolved to ensure smooth pipeline execution~.
- There is no impact on the existing F5 Integration. As discussed over email, the current pipeline implementation will not affect the access logs.

### CODE REVIEW

File: f5-distributed-cloud/assets/logs/f5-distributed-cloud-services.yaml [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: Map `dst_port` to `network.destination.port`
+          enabled: true
+          sources:
+            - dst_port
+          sourceType: attribute
+          target: network.destination.port
+          targetType: attribute
+          preserveSource: true
```
[the remark this answers is not in the document]
Comment (bhargavnariyanicrest) [reply f5-distributed-cloud/assets/logs/f5-distributed-cloud-services.yaml:178]: Yes, this is an update integration and we would like to preserve the older values for anyone using the exisiting attributes in their custom created assets.

[the remark this answers is not in the document]
Comment (mathieu-charbonnel) [reply f5-distributed-cloud/assets/logs/f5-distributed-cloud-services.yaml:178]: Ok thank you

**Comment by mathieu-charbonnel** [comment]
It does not seem possible for me to add the label required to test on staging: [assets/deploy-logs-staging]

**Review by jnhunsberger — APPROVED** [review]
Approved.

**Review by mathieu-charbonnel — APPROVED** [review]

**Review by moreauowen — APPROVED** [review]

**Review by thibaultkrebs — APPROVED** [review]

**Pull request merged**